docs/modules/adc.md

```diff
@@ -33,13 +33,13 @@ Configuration ADC1 capture attenuation of channels
 - `adc_number` Only `adc.ADC1` now
 - `channel`  When using `adc.ADC1`: `0` to `7`. 0: GPIO36, 1: GPIO37, 2: GPIO38, 3: GPIO39, 4: GPIO32, 5: GPIO33, 6: GPIO34, 7: GPIO35
 - `atten` One of following constants
-  - `adc.ATTEN_0db`    The input voltage of ADC will be reduced to about 1/1    (1.1V when VDD_A=3.3V)
-  - `adc.ATTEN_2_5db`  The input voltage of ADC will be reduced to about 1/1.34 (1.5V when VDD_A=3.3V)
-  - `adc.ATTEN_6db`    The input voltage of ADC will be reduced to about 1/2    (2.2V when VDD_A=3.3V)
-  - `adc.ATTEN_11db`   The input voltage of ADC will be reduced to about 1/3.6  (3.9V when VDD_A=3.3V,  maximum voltage is limited by VDD_A)
+    - `adc.ATTEN_0db`    The input voltage of ADC will be reduced to about 1/1    (1.1V when VDD_A=3.3V)
```

Comment:
I think this will break things. We just had a PR doing it the other way around and it looked good afterwards.
See #3425

Reply:
I have changed two spaces in unordered list to 'tab'. Everything is rendered correctly now.

Reply:
> I think this will break things. We just had a PR doing it the other way around and it looked good afterwards.
> See #3425

That was about a missing empty/new line before the start of the list AFAICS.

Reply:
> I have changed two spaces in unordered list to 'tab'

Wouldn't four spaces be safer?

Reply:
Of course, four spaces is better. This was done in the previous commit.